cli/src/commit_templater.rs

```diff
@@ -2197,3 +2220,41 @@ pub fn builtin_cryptographic_signature_methods<'repo>(
     );
     map
 }
+
+#[derive(Debug, Clone)]
+pub struct AnnotateData {
```

Comment:
nit: suppose we'll add line content filed, `AnnotationLine` or `AnnotatedLine` might be a better name.